src/scaffoldfitter/utils/zinc_utils.py

```diff
@@ -0,0 +1,401 @@
+"""
+Utility functions for easing use of Zinc API.
+"""
```

Comment:
All of this should be added to Zinc utils.

Reply:
Will do... may leave a couple of functions that aren't very reusable.

Reply:
Not ideal having a single file zinc.py in opencmiss.utils. Would be better to have a zinc folder/module with utilities for specific parts e.g. zinc.field for field-specific utilities. Is it feasible to re-org in this way at this time; increment version number etc. Or should I just add to zinc.py?

Reply:
If it needs the re-org now then now is the time :)